### DISCUSSION TITLE
Feature request: supporting build.gradle / build.gradle.kts

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [community/community#21674] [New to GitHub]
<p>(if this topic is on wrong forum, please move, thanks!)</p>

**Comment by BelHFL** [comment, marked as answer]
<p>Hi <a class="mention" href="/kisaragieffective">@kisaragiEffective</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>
